perfkitbenchmarker/providers/ibmcloud/util.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2020 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with IBM Cloud resources."""
+
+import os
+import json
+import yaml
+import dataclasses
+
+from typing import Any, List, Dict
+
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import os_types
+from perfkitbenchmarker.providers.ibmcloud import ibmcloud
+
+WINDOWS = os_types.WINDOWS  # all windows
+DEBIAN = os_types.DEBIAN  # all debian
+UBUNTU = 'ubuntu'  # all ubuntu
+REDHAT = 'redhat'  # all redhat
+CENTOS = 'centos'  # all centos
+UNKNOWN = 'unknown'
+
+SUBNET_SUFFIX = 's'
+DELIMITER = 'z-z'
+
+# these are used to create extra subnets for multi vnic support in perfkit,
+# not used for vm provisioning, up to 4 extra totaling 5 subnets
+SUBNET_SUFFIX_EXTRA = 'sxs'
```

Comment:
Seems odd to have "SUBNET_SUFFIX_EXTRA" in a util.py class.  Is there an ibmcloud_network.py file that this can be put into?

Reply:
yes, moved.